code/ContactList.java

```diff
@@ -0,0 +1,29 @@
+public class ContactList {
+
+    static final int DEFAULT_CAPACITY = 10;
+
+    int capacity;
+    int friendCount;
+    Friend[] friends;
```

Comment:
`capacity` is presumably always `friends.length` -- topic2 even [covers the length field](https://github.com/jameshughes89/cs102/blob/main/source/topic2.rst#arrays). Maybe better to remove the field.  
(package private) -> `private` fields?  
Similar comment applies to code blocks in the notes themselves.

Reply:
Good call. Artifact of cpp.